### PR TITLE
1930 - Fix Datagrid short row height on chart target

### LINKED ISSUE #1930: Datagrid: Row heights aren't set correctly on targeted achievement formatter
**Describe the bug**

When using the targetted achivement formatter the row heights will extend over the expended row height, this is very noticible on frozen columns since the two sections will not align.

**To Reproduce**
1. See the attached example
2. Open attached example
3. Click on the toolbar ... menu and select short row height
4. See rows are no longer aligned

Can also be seen simply on 
http://latest-enterprise.demo.design.infor.com/components/datagrid/test-targeted-achievement.html
- change row height and that the rows do not change size.

**Expected behavior**
expect targeted achievement chart to fit in short rowheight

**Version**
- ids-enterprise: v4.17.0

**Screenshots**
image

**Platform**
 - Windows 10
 - Chrome Browser Version 73.0.3683.86 (Official Build) (64-bit)

**Additional context**
Sample:
```
<div class="row">
  <div class="twelve columns">

    <div id="datagrid">
    </div>

  </div>
</div>

<script>
  var gridApi = null;

  $('body').one('initialized', function () {

      var grid,
        data = [],
        columns = [];

      //Define Columns for the Grid.
      columns.push({ id: 'manufacturerName', name: 'Manufacturer Name', field: 'manufacturerName', filterType: 'text'});
      columns.push({ id: 'inventoryValue', name: 'Inventory', field: 'inventoryValue', width: 100});
      columns.push({ id: 'customerSatisfaction', name: 'With Primary Color', field: 'customerSatisfaction', formatter: Formatters.TargetedAchievement, showPercentText: true , width: 200});
      columns.push({ id: 'customerSatisfaction2', name: 'With Alert Colors', field: 'customerSatisfaction', formatter: Formatters.TargetedAchievement, ranges: [{'min': 0, 'max': 40, 'classes': 'error'}, {'value': 79, 'classes': 'good'}] , width: 200});
      columns.push({ id: 'customerSatisfaction3', name: 'To a Target', field: 'customerSatisfaction', formatter: Formatters.TargetedAchievement, target: 80, width: 200});
      columns.push({ id: 'customerSatisfaction3', name: 'Custom Text', field: 'customerSatisfaction', formatter: Formatters.TargetedAchievement, text: 'Custom Text <% percent %>', tooltip: 'Custom Tooltip', width: 200});
      columns.push({ id: 'gap', name: ''});

      //Get some data via ajax
      var url = '{{basepath}}api/bikes';

      $.getJSON(url, function(res) {

        $('#datagrid').datagrid({
          columns: columns,
          dataset: res,
          paging: true,
          pagesize: 10,
          enableTooltips: true,
          frozenColumns: {
            left: ['manufacturerName', 'inventoryValue'],
          },
          toolbar: {title: 'Bike Dealers', results: true, personalize: true, actions: true, rowHeight: true}
        });
      });

  });

</script>
```

Css I changed to make it work in landmark:
```
  .datagrid.short-rowheight {
    .chart-completion-target {
      height: 21px !important;
      top: 0px !important;

      .target {
        height: 21px !important;
        margin-top: 0px !important;
      }
      .completed {
        height: 21px !important;
        margin-top: -21px !important;
        margin-bottom: 0px !important;
      }
      .chart-targeted-text {
        margin-top: -21px !important;
      }
    }
  }
```

### PR DESCRIPTION
**Pull request by davidcarlsonberg**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Short row height columns were not adjusting properly on `.chart-targeted-achievement` and `.chart-completion-target`.

**Related github/jira issue (required)**:
Closes #1930.

**Steps necessary to review your pull request (required)**:
- Pull branch
- Run app
- Visit http://localhost:4000/components/datagrid/test-targeted-achievement.html
  - Switch Row Height via `...` button.
  - Ensure rows are properly aligned.

### CODE REVIEW

File: src/components/datagrid/_datagrid.scss [hunk 1]
```diff
@@ -948,6 +948,8 @@ $datagrid-short-row-height: 23px;
         margin-top: -33px;
       }
     }
+
+
```
Comment (tmcconechy) [inline src/components/datagrid/_datagrid.scss]:
```suggestion
```

Comment (tmcconechy) [reply src/components/datagrid/_datagrid.scss]: Removed extra line (i think)

**Comment by deep7102** [comment]
All Looks good, one thing saw
it should now display empty tooltip (see middle columns `With Alert Colors, To a Target`)

**Comment by davidcarlsonberg** [comment]
> All Looks good, one thing saw
> it should not display empty tooltip (see middle columns `With Alert Colors, To a Target`)

Probably should create a new issue for this, yes?

**Comment by tmcconechy** [comment]
@davidcarlsonberg sure you can punt that. I think that it should somehow show the value even though there is no text for it. Maybe adding an audble span would do it?

**Pull request merged**